rmgpy/tools/canteramodel.py

```diff
@@ -370,8 +370,8 @@ def plot(self, data, top_species=10, top_sensitive_reactions=10, top_sensitive_s
                     os.path.join(self.output_directory, '{0}_{1}_reaction_sensitivity.png'.format(i + 1, species.to_chemkin())))
                 if self.thermo_SA:
                     ThermoSensitivityPlot(x_var=time,
-                                          y_var=thermodynamic_sensitivity_data[j * num_ct_species:(j + 1) * num_ct_species],
-                                          xlabel='dln(c)/d(H_i) [(J/mol)^-1]',
+                                          y_var=thermodynamic_sensitivity_data[j * num_ct_species:(j + 1) * num_ct_species]*4184,
```

Comment:
I think the conversion factor is 4184000

Reply:
Ah sorry I'll update this now. Thanks for catching that